ggshield/cmd/main.py

```diff
@@ -140,6 +148,11 @@ def cli(
     logger.debug("args=%s", sys.argv)
     logger.debug("py-gitguardian=%s", pygitguardian.__version__)
 
+    if check_for_updates and "PYTEST_CURRENT_TEST" not in os.environ:
```

Comment:
Can you add a comment explaining why we added the `PYTEST_CURRENT_TEST` check? Should help future-us being less puzzled about this code.